packages/engine/src/config/experiment.rs

```diff
@@ -56,11 +56,11 @@ impl Config {
 
         let run = Arc::new(experiment_run);
 
-        // TODO: Rust, Python
-        // TODO: Ask packages for what language execution they require.
+        // TODO: Rust
+        // TODO: Ask packages for what language execution they require. Done for Python.
```

Comment:
Could you do that for JavaScript and Rust as a follow up? For `Rust` please ping me if there are any issues as the runner is not implemented. We also want a panic if it's trying to be started.